services/tenant-ui/frontend/src/main.ts

```diff
@@ -51,6 +51,22 @@ async function loadApp() {
   // manually remove the listener
   unsubscribe();
 
+  // Matomo Setup
+  const MATOMO_URL: string = configStore.config.frontend.matomoUrl;
+  if (MATOMO_URL) {
+    import('./matomoSetup')
+      .then((m: { setup: (url: string) => void }) => {
+        console.log(m.setup(MATOMO_URL));
+        console.log('initialized Matomo');
+        console.log(`${MATOMO_URL}`);
+      })
+      .catch((error) => {
+        console.log(error);
```

Comment:
Should log errors with `console.error`, I think, if we're not handling the error and just logging it